airgun/entities/task.py

```diff
@@ -20,6 +22,35 @@ def read(self, entity_name, widget_names=None):
         view = self.navigate_to(self, 'Details', entity_name=entity_name)
         return view.read(widget_names=widget_names)
 
+    def set_chart_filter(self, chart_name, index=None):
+        """Remove filter from searchbox ans set filter from specific chart
+
+        :param index: index in 'StoppedChart' table,
+            dict with 'row' number and 'focus' as column name
+        """
+        view = self.navigate_to(self, 'All')
+        chart = getattr(view, chart_name)
+        view.searchbox.clear()
+        if chart_name == 'StoppedChart' and index:
+            chart.table[index['row']][index['focus']].click()
+        else:
+            chart.name.click()
+
+    def read_widget(self, widget_name):
+        """Read specific widget"""
+        view = self.navigate_to(self, 'All')
+        return getattr(view, widget_name).read()
+
+    def table_row_count(self):
```

Comment:
There is [`Pagination` widget](https://github.com/SatelliteQE/airgun/blob/master/airgun/widgets.py#L1487). Could you extend it to include `total_items` widget, and return it's content on `read()`? Then on test side you should be able to do:
```python
pagination = session.task.read_all('pagination')
assert pagination['total_items'] == some_oracle
```